cmd/cli/kubectl-kyverno/utils/common/common.go

```diff
@@ -608,8 +607,7 @@ func GetPoliciesFromPaths(fs billy.Filesystem, dirPath []string, isGit bool, pol
 			}
 			policiesFromFile, errFromFile := utils.GetPolicy(policyBytes)
 			if errFromFile != nil {
-				err := fmt.Errorf("failed to process : %v", errFromFile.Error())
-				errors = append(errors, err)
```

Comment:
Why not append to error list anymore?

Reply:
We don't do anything with `errors` in the remaining code